packages/main/src/services/http/server/schema.ts

```diff
@@ -11,6 +11,7 @@ export const getSettingsSchema = {
       option: {
         type: 'string',
         enum: settingsConfig
+          // I'm not sure about this RESTRICTED_SETTINGS is updated in somewhere
           .filter(({ name }) => !RESTRICTED_SETTINGS.includes(name as never))
```

Comment:
I need help with this.

Reply:
This, as well as READONLY_SETTINGS, was made so that we're safe in the future - currently has no effect but we should keep it.